include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -155,7 +203,11 @@ namespace SuperiorMySqlpp
             }
 
         public:
-
+            /**
+             * Returns metadata for the statement result.
+             * There metadata are retrieveable even before execute().
```

Comment:
Are you sure they are available before execute()?

Reply:
According to MySql documentation, yes. It calls `mysql_stmt_result_metadata` that fetches this information independently on `execute()`, see example at https://dev.mysql.com/doc/refman/5.7/en/mysql-stmt-fetch.html.